.gitlab-ci.yml

```diff
@@ -88,3 +89,20 @@ deploy-fallback-only:
   extends: .kubectl
   script:
     - ./deploy/deploy.sh HX # fallback
+
+lightouse:
+  image: cypress/browsers:node14.15.0-chrome86-ff82
+  only:
+    refs:
+      - main
```

Comment:
Are lighthouse tests expensive? If so do we want to run them at some regular interval rather than on each push to main? If so, we could create a branch `lighthouse` and push to this whenever we want a test or maybe [schedule it](https://docs.gitlab.com/ee/ci/pipelines/schedules.html).

Reply:
I think the only thing we'd test by running them regularly is if the API is down, which is not really what we're supposed to test for frontend, they have their own monitoring.
Associating lighthouse runs to specific commits should make it easier to track when (as in "in which specific commit") we added a library that slowed down the page for example.
Also, eventually, I'd like to have something like this set-up https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/server.md Which works better when associated with specific commits

Reply:
I'm sure we can know which commit a scheduled lighthouse test was run against. We would however lose specificity because we will then have a set of commits that could have caused a loss in performance (or the server health at that point in time).